extensions/policy-referring-connector/src/main/java/de/sovity/edc/extension/policy/functions/AbstractReferringConnectorValidation.java

```diff
@@ -134,7 +124,22 @@ private boolean isReferringConnector(
      * @return true if claim equals the referringConnector
      */
     private static boolean isAllowedReferringConnector(
-            String referringConnectorClaim, String referringConnector) {
-        return referringConnectorClaim.equals(referringConnector);
+            String referringConnectorClaim, Object referringConnector, Operator operator) {
+        if (operator == Operator.IN) {
+            var referringConnectorList = (List<?>) referringConnector;
+            return referringConnectorList.contains(referringConnectorClaim);
+        } else if (operator == Operator.EQ) {
+            //support comma separated lists here as well
+            if (referringConnector instanceof String string) {
+                var referringConnectorList = string.split(",");
+                for (var connector : referringConnectorList) {
```

Comment:
May be simplified to
```suggestion
               referringConnectorString.split(",").stream().anyMatch(participantId-> participantId.equals(referringConnectorClaim));
```